### PR TITLE
nopoll 0.4.5 (new formula)

### PR DESCRIPTION
**Pull request by lieuwex**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This commit adds the nopoll formula to the tap.

I tried to get the HEAD version working, but that was harder than
expected.

### CODE REVIEW

File: Formula/nopoll.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Nopoll < Formula
+  desc "OpenSource C WebSocket toolkit"
+  homepage "https://www.aspl.es/nopoll/"
+  url "https://www.aspl.es/nopoll/downloads/nopoll-0.4.5.b375.tar.gz"
+  sha256 "4ecbd277714c9acd154277a46388689f9d651836ca5c69990e35d1eee8c7ceae"
+  head "https://github.com/asples/nopoll.git"
+
+  depends_on "openssl"
+
+  depends_on "autoconf" => :build
```
Comment (DomT4) [inline Formula/nopoll.rb]: Are you sure these are required for the stable build? Looking at the stable tarball it seems unlikely.

Comment (lieuwex) [reply Formula/nopoll.rb]: True, that's a leftover from trying to build HEAD. I will remove it.

**Comment by lieuwex** [comment]
@DomT4 Could you take a look again after my last change?

**Review by ilovezfs — APPROVED** [review]

**Review by DomT4 — APPROVED** [review]

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Congratulation @lieuwex on your first contribution to Homebrew!

**Comment by lieuwex** [comment]
Thank you! 😊

Thank you guys too for adding stuff I forgot.

**Pull request closed without merging**